### PR TITLE
Fix #234 (Improve Example)

### LINKED ISSUE #234: Label example in Section 6.1
The first example in Section 6.1 shows a credential, not a type metadata document, but is not labelled as such.

The example in question:
```
{
  "vct": "https://betelgeuse.example.com/education_credential",
  "vct#integrity": "sha256-WRL5ca_xGgX3c1VLmXfh-9cLlJNXN-TsMk-PmKjZ5t0",
  ...
}
```

### PR DESCRIPTION
**Pull request by danielfett**
Fixes #234

### CODE REVIEW

**Review by bc-pi — APPROVED** [review]

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 1]
```diff
@@ -500,21 +500,23 @@ Applications using Type Metadata defined in this specification are called "Consu
...
 
-Type Metadata about the value of a `vct` claim value can be retrieved as described in (#retrieving-type-metadata).
+The following is an example of an SD-JWT VC payload, containing a `vct` claim
+with the value `https://betelgeuse.example.com/education_credential`:
 
 ```json
 {
   "vct": "https://betelgeuse.example.com/education_credential",
-  "vct#integrity": "sha256-WRL5ca_xGgX3c1VLmXfh-9cLlJNXN-TsMk-PmKjZ5t0",
+  "vct#integrity": "sha256-oTxxE//FNomPebw6yH9aNuEPUjbiYqaxBj1ZgMMBsxHmjTOBDhRiVCUD2bhVlzYg",
```
Comment (bc-pi) [inline draft-ietf-oauth-sd-jwt-vc.md:509]: wait... `oTxxE//FNomPebw6yH9aNuEPUjbiYqaxBj1ZgMMBsxHmjTOBDhRiVCUD2bhVlzYg` decodes to 48 bytes, which doesn't track too well with `sha256` while `WRL5ca_xGgX3c1VLmXfh-9cLlJNXN-TsMk-PmKjZ5t0` is the 32 bytes one would expect. Why'd you change this value?

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 2]
```diff
@@ -526,6 +528,10 @@ The following is an example for a Type Metadata document:
 }
 ```
 
+Note: The hash of the Type Metadata document shown in the second example must be equal
+to the one in the `vct#integrity` claim in the SD-JWT VC payload,
+`oTxxE//FNomPebw6yH9aNuEPUjbiYqaxBj1ZgMMBsxHmjTOBDhRiVCUD2bhVlzYg`.
```
Comment (bc-pi) [inline draft-ietf-oauth-sd-jwt-vc.md:533]: see also https://github.com/oauth-wg/oauth-sd-jwt-vc/pull/235/files#r1633871626

**Review by bc-pi — CHANGES_REQUESTED** [review]
how many bytes would a vct#integrity byte if a vct#integrity could byte integrity?

**Review by awoie — CHANGES_REQUESTED** [review]
looks good to me but @bc-pi comment has to be fixed

File: draft-ietf-oauth-sd-jwt-vc.md (hunk 1 above)
Comment (bc-pi) [reply draft-ietf-oauth-sd-jwt-vc.md:509]:
```suggestion
  "vct#integrity": "sha256-WRL5ca_xGgX3c1VLmXfh-9cLlJNXN-TsMk-PmKjZ5t0",
```

File: draft-ietf-oauth-sd-jwt-vc.md (hunk 2 above)
Comment (bc-pi) [reply draft-ietf-oauth-sd-jwt-vc.md:533]:
```suggestion
`WRL5ca_xGgX3c1VLmXfh-9cLlJNXN-TsMk-PmKjZ5t0`.
```

**Review by bc-pi — APPROVED** [review]

**Review by awoie — APPROVED** [review]

**Pull request merged**